content/en/docs/release-notes/release-notes-1.7.md

```diff
@@ -7,12 +7,117 @@ type: "docs"
 
 ## Breaking Changes (You **MUST** read this before you upgrade!)
 
+⚠ Following their deprecation in version 1.5, the cert-manager API versions v1alpha2, v1alpha3, and v1beta1 have been removed.
+You must ensure that all cert-manager custom resources are stored in etcd at version v1
+and that all cert-manager `CustomResourceDefinition`s have only v1 as the stored version.
+
+Since release 1.7, `cmctl` can automatically migrate any deprecated API resources.
+Please download `cmctl-v1.7.0-beta.0` and read [Removing Deprecated API Resources]
```

Comment:
suggestion: if these are the actual draft release notes for 1.7, should we refer to `cmctl-v1.7.0` rather than `cmctl-v1.7.0-beta.0`?

also, could we link to the release page (https://github.com/jetstack/cert-manager/releases/tag/v1.7.0-beta.0 / https://github.com/jetstack/cert-manager/releases/tag/v1.7.0)?

```suggestion
Please download `cmctl-v1.7.0` from [here](https://github.com/jetstack/cert-manager/releases/tag/v1.7.0) and read [Removing Deprecated API Resources]
```